doc/operator-capabilities.md

```diff
@@ -48,29 +48,33 @@ The Operator offers the following basic features:
 
 3. Can you set operand configuration in the CR? If so, what configuration is supported for each operand?
 
-4. Does the managed application / workload get updated in a non-disruptive fashion when the configuration of the CR is changed?
+4. Can you override the operand images through the CR or an env var?
 
-5. Does the status of the CR reflect that configuration changes are currently applied?
+5. Does the managed application / workload get updated in a non-disruptive fashion when the configuration of the CR is changed?
 
-6. What additional operand configuration could still be added?
+6. Does the status of the CR reflect that configuration changes are currently applied?
 
-7. Do all of the instantiated CRs include a status block? If so, does it provide enough insight to the user about the application state?
+7. What additional operand configuration could still be added?
 
-8. Do all of your CRs have documentation listing valid values and mandatory fields?
+8. Do all of the instantiated CRs include a status block? If so, does it provide enough insight to the user about the application state?
+
+9. Do all of your CRs have documentation listing valid values and mandatory fields?
+
+10. If your operator includes a CSV, are all images used listed in the CSV under relatedImages?
 
 ---
 
 ## Level 2 - Seamless Upgrades
 
-The Operator offers the following features related to upgrades:
+Seamless upgrades mean the upgrade is as easy as possible for the user. You should support seamless upgrades of both your operator and operand, these would normally go hand in hand, an upgrade of the operator would automatically ensure the instantiated resources for each CR are in the new desired state and which would upgrade your operand. Upgrade may also be defined multiple ways, such as updating the software of the operand - and other internals specific to the application - such as schema migrations. It should be very clear what is upgraded when this takes place, and what is not.
```

Comment:
The upgrade of the Operator and Operand don't necessarily go hand-in-hand though. This is just one option. Many Operators control the Operand version based on a property in the CR. The `Seamless Upgrade` capability would recommend that the Operator is capable of upgrading the Operand either in this manner or automatically to the version the Operator prefers.

Reply:
@dmesser I agree with you, not all operators have a 1-1 relationship between operator and operand. I am under the impression this is still a best practice though if it is possible to achieve with your operator and operand.

This paragraph does call out that this is the normal case but there are other defined upgrades also.